orc8r/gateway/python/magma/magmad/config_manager.py

```diff
@@ -82,6 +82,7 @@ def process_update(self, stream_name, updates, resync):
                     since config is contained in one DB element, hence all
                     data is sent in every update.
         """
+        shared_mconfig = 'shared_mconfig'
```

Comment:
Let's move this up to the top of the file as other consts are defined that way.
See https://github.com/magma/magma/blob/master/orc8r/gateway/python/magma/magmad/config_manager.py#L29